processor/examples/README.md

```diff
@@ -0,0 +1,48 @@
+## Getting started with samples ##
+
+## Building the library
+- Clone the repo and cd to the repo directory
+```bash
+git clone https://github.com/Azure/azure-event-hubs-node.git
+cd client
+```
+- Install typescript, ts-node globally (required for running the samples)
+```bash
+npm i -g typescript
+npm i -g ts-node
+```
+- NPM install from the root of the package
+```bash
+npm i
+```
+- Build the project
+```bash
+npm run build
+```
+
+## Before executing a sample
+- Go to the [Azure Portal](https://portal.azure.com).
+- Here are the docs which would help you create a event hubs and eventhub/iothub resources in the portal :
+  - [Azure Event Hubs - NodeJS DOCS](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-node-get-started-send).
+  - [Azure IoT Hubs - NodeJS DOCS]
+- In the portal, go to **Dashboard > Event Hubs > _your-eventhubs-namespace_**.
+- Note down the "Primary Connection String" of **RootManageSharedAccessKey** at **Shared access policies** in the **Settings** section.
+- Note down the name of the event hub that you created
+> _Note : **RootManageSharedAccessKey** is automatically created for the namespace and has permissions for the entire namespace. If you want to use restricted access, refer [Shared Access Signatures](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-sas), create the Access Keys exclusive to the specific created Queue/Topic._
+- You would also need an Azure Storage Account for all the `*Eph.ts` examples. The docs are at
+[Create a storage account](https://docs.microsoft.com/en-us/azure/storage/common/storage-quickstart-create-account?tabs=azure-portal). After the storage account is created, find the connection string at **Access keys** tab in the **Settings** section.
+Before running a sample, update it with the connection string and the hub name you have noted down above.
+- to load environment variables from an `.env` file, please refer to [NPM dotenv package](https://www.npmjs.com/package/dotenv) for more details
```

Comment:
@ShivangiReja Can you take a look at the above instructions and give any feedback you have?